ogc-example/src/main/scala/geotrellis/server/ogc/conf/StyleConf.scala

```diff
@@ -36,10 +36,10 @@ final case class ColorRampConf(
   stops: Option[Int],
   minRender: Option[Double],
   maxRender: Option[Double],
+  clampWithColor: Boolean = false,
```

Comment:
An alternative to this single option would be two independent additional options:
- `maxRenderColor: Int = 0x00000000`
- `minRenderColor: Int = 0x00000000`

Which would serve the same purpose, but require that the user inspect the color ramp for the default case where the user wants the min/max colors to match those of the ramp.

In any case, the implementation here is explicitly what's requested in the issue.